setup.py

```diff
@@ -88,10 +85,6 @@
               "example\n"
         f.write(msg)
 
-DATA_DIR = os.path.dirname(data.__file__)
```

Comment:
`data_files` argument is processed by `setup.py` when `install` command is used but not processed when `develop` command is used.
`pip install` internally uses `install` command. `pip install -e` internally uses `develop` command.
So if the package has been installed using `pip install -e` then the user has to run `python setup.py install_data` to install data files.